src/main/java/net/atos/client/zgw/zrc/ZRCClientService.java

```diff
@@ -90,19 +90,19 @@ public Rol<?> createRol(final Rol<?> rol) {
      * @return Created {@link Rol}.
      */
     public Rol<?> createRol(final Rol<?> rol, final String toelichting) {
-        zgwClientHeadersFactory.setAuditToelichting(toelichting);
+        zgwClientHeadersFactory.setAuditToelichting("%s:%s".formatted(rol.getOmschrijving(), toelichting));
         return zrcClient.rolCreate(rol);
     }
 
     /**
      * Delete {@link Rol}.
      *
-     * @param rolUUID     uuid van de {@link Rol}/
+     * @param rol         de betreffende rol {@link Rol}/
      * @param toelichting de toelichting
      */
-    public void deleteRol(final UUID rolUUID, final String toelichting) {
-        zgwClientHeadersFactory.setAuditToelichting(toelichting);
-        zrcClient.rolDelete(rolUUID);
+    public void deleteRol(final Rol<?> rol, final String toelichting) {
+        zgwClientHeadersFactory.setAuditToelichting("%s: %s".formatted(rol.getOmschrijving(), toelichting));
```

Comment:
Moet hier niet omschrijvingGeneriek worden gebruikt i.p.v. omschrijving?

Reply:
Denk het niet, dat is de "systeem" naam. omschrijving is de naam die de beheerder heeft toegekend aan deze naam.